github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1906,3 +1906,39 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: false
+    annotations:
+      fork-per-release: "true"
+    cluster: ibm-prow-jobs
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    labels:
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "false"
+      preset-docker-mirror-proxy: "false"
+      preset-podman-in-container-enabled: "true"
+      preset-shared-images: "true"
+      rehearsal.allowed: "true"
+    max_concurrency: 11
+    name: pull-kubevirt-metrics-lint
+    run_if_changed: docs/metrics.md
+    optional: true
+    skip_branches:
+      - release-\d+\.\d+
+    spec:
+      containers:
+        - command:
+            - /usr/local/bin/runner.sh
+            - /bin/sh
+            - -c
+            - |
+              make lint-metrics
+          image: quay.io/kubevirtci/golang:v20230607-07930d7
```

Comment:
IPv6 is not really needed with this linter.
could you please try add the following env var:
```
        env:
        - name: PODMAN_IN_CONTAINER_IPV6_ENABLED
          value: "false"
```
?